src/wp-color-picker-alpha.js

```diff
@@ -20,8 +20,16 @@
 		_after = '<div class="wp-picker-holder" />',
 		_wrap = '<div class="wp-picker-container" />',
 		_button = '<input type="button" class="button button-small" />',
-		// Prevent CSS issues in < WordPress 4.9
-		_deprecated = ( wpColorPickerL10n.current !== undefined );
+		_deprecated = false;
```

Comment:
@prince5ingh All the variables are comma-separated. This will be better in my opinion:

```
_deprecated = false,
__ = wp.i18n.__;
```